youtube_dl/extractor/vier.py

```diff
@@ -67,14 +73,28 @@ def _real_extract(self, url):
         self._sort_formats(formats)
 
         title = self._og_search_title(webpage, default=display_id)
-        description = self._og_search_description(webpage, default=None)
         thumbnail = self._og_search_thumbnail(webpage, default=None)
+        description = self._html_search_regex(
+            r'''(?x)<div\ class="[^"]*field-type-text-with-summary[^"]*">\s*
+                      (?:<div\ class="[^"]+">\s*)*
+                     <p>\s*(?:<span>)?(.+?)</''',
+            webpage, 'description', default=None)
+        episode_number = int_or_none(self._search_regex(
+            r'(?i)aflevering (\d+)', title, 'episode_number', default=None,
+            fatal=False))
+        upload_date = unified_strdate(self._html_search_regex(
+            r'''(?x)<div\ class="[^"]*field-name-post-date[^"]*">\s*
```

Comment:
Don't you have to escape whitespace in a triple quote? I do know that without the two backslashes the regexp doesn't match.